### ISSUE TITLE
Next Major version should be v1.0.0-next-major.1

### ISSUE THREAD

**Issue by cletustboone** [rdkcentral/firebolt-core-sdk#74]
semantic-release determined it to be [v0.10.0-next-major.1](https://github.com/rdkcentral/firebolt-core-sdk/releases/tag/v0.10.0-next-major.1) because there is no [breaking change commit](https://semantic-release.gitbook.io/semantic-release/#commit-message-format) in [the history for the next-major branch](https://github.com/rdkcentral/firebolt-core-sdk/commits/next-major).

See the semantic-release docs on [how to create a commit that will trigger a breaking change](https://semantic-release.gitbook.io/semantic-release/#commit-message-format) or new major version.

For example, you need a commit in the `next-major` history that looks like this:
```
feat: A feature that changes the customer-facing API in a non-additive way

BREAKING CHANGE: A new feature was added that will allow you (the API consumer) to do something amazing. This new feature replaces OLD feature, XYZ. The feature's API is removed from firebolt.
```
So see, the commit message, `feat: ...`, can be anything, but if you have `BREAKING CHANGE: ...` in the _commit ~body~ **FOOTER**_, then semantic-release understands that as a breaking change and will increment the major version accordingly.

**Comment by cletustboone** [comment]
Anyone with write access to `next-major` can just find the commit that introduced the breaking change, edit the commit message to include the required text, then push that back to the branch, then re-run the release automation and it should generate the correct release. Then just delete the errant one on the releases tab.

**Comment by kevinshahfws** [comment]
That's interesting... However, [commit message format](https://kapeli.com/cheat_sheets/Semantic_Commits.docset/Contents/Resources/Documents/index) doesn't have BREAKING CHANGE: as type.

Should it be **_feat_** in that case??

**Comment by cletustboone** [comment]
Fixed by a30c424442f95015def0ff279b3363383ba0c73b, thanks!
